src/js/view/view.js

```diff
@@ -572,7 +572,7 @@ function View(_api, _model) {
             _model.set('height', playerHeight);
         }
 
-        if (_floatingConfig && _resizeOnFloat) {
+        if (_floatingConfig && _resizeOnFloat && playerStyle.width !== _model.get('containerWidth')) {
```

Comment:
Do you want to style `_playerElement` when `playerStyle.width === _model.get('containerWidth')`? If not, don't add this with `&&`.